src/components/ProductList.tsx

```diff
@@ -60,6 +62,8 @@ const ProductList = ({
         <div
           key={product.id}
           ref={i === products.length - 1 ? lastRef : undefined}
+          onClick={() => onProductClick && onProductClick(product.id)}
```

Comment:
```suggestion
          onClick={() => onProductClick?.(product.id)}
```

`?.` optional operator는 함수에도 적용할 수 있습니다.